src/main/java/org/junit$pioneer/jupiter/TempDirectory.java

```diff
@@ -168,7 +167,7 @@ public static TempDirectory createInDefaultDirectory() {
 	}
 
 	/**
-	 * Returns a {@code TempDirectory} extension that uses the supplied
+	 * Create a new {@code TempDirectory} extension that uses the supplied
```

Comment:
Is there a good reason to promise a fresh instance? I prefer the contract of static factory methods to be purposely vague about what exactly is returned. This way there is more room to refactor.

Reply:
Agreed, I changed it back. Ready to be merged now IMHO. 🙂